### PR TITLE
add docs for `signTransactions`

### PR DESCRIPTION
**Pull request by amityadav0**

### CODE REVIEW

File: docs/guide/execute-transaction.md [hunk 1]
```diff
@@ -212,4 +212,18 @@ const transaction: wtypes.TransactionAndSign = {
...
+
+```
+const transaction: wtypes.TransactionAndSign = [{
+  transaction: SDKTx,
+  sign: {sign: wtypes.SignType.SecretKey, fromAccount: bobAcc}
+}]
+const rawSign = signTransactions(transaction)
+```
+
+`rawSign` has array of raw signed transactions.
```
Comment (ratik21) [inline docs/guide/execute-transaction.md:229]: Could you please add an example in a run script (maybe in `/asa`) ? And link it here?

File: docs/guide/execute-transaction.md [hunk 2]
```diff
@@ -212,4 +212,18 @@ const transaction: wtypes.TransactionAndSign = {
 }
 
 const res = await executeTransaction(deployer, transaction);
-```
\ No newline at end of file
+```
+
+## SignTransactions function
+
+This function takes array of `TransactionAndSign` objects and returns raw signed transaction
```
Comment (ratik21) [inline docs/guide/execute-transaction.md:219]: Please add a link to `TransactionAndSign` type.

**Review by ratik21 — APPROVED** [review]
pre-approving

File: docs/guide/execute-transaction.md (hunk 1 above)
Comment (amityadav0) [reply docs/guide/execute-transaction.md]: ok

**Pull request merged**